### DISCUSSION TITLE
Backup & Restore (Data & Postgresql Database)

### DISCUSSION THREAD

**Discussion by DBRNinja** [requarks/wiki#3084] [Help / Questions]
### Real behavior

After installing a new instance, on a new Debian 10 VM, importing the old Postgresql database and data, the nodejs wikijs server does not start ...

Error in the "migrations" table
![Sans titre](https://user-images.githubusercontent.com/79094402/107954956-72b5d580-6f9d-11eb-8f24-40ad0ff1d1d2.png)

### Steps to reproduce the behavior

1- Install an instance of WikiJS 2.5.170
2- Create a Postgresql database and import the wikijs.dump from the old database to the new one
3- Import the data into the data folder (manual.backup.tar.gz)
 4- Start the wikijs server

**Comment by NGPixel** [comment]
Looks like an incomplete or corrupted backup from your side. Also, you mention 2.5.170 but the logs state 2.5.159. Make sure you're using the latest version.
